### PR TITLE
Feature 1309 - adding the secret for idam

### PR DESCRIPTION
**Pull request by sampankumar**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Review by laarmada — APPROVED** [review]

File: src/main/app/client/civilServiceClient.ts [hunk 1]
```diff
@@ -28,10 +28,12 @@ export class CivilServiceClient {
   async retrieveByDefendantId(req: AppRequest): Promise<Claim[]> {
     const config = this.getConfig(req);
     let claims : Claim[] = [];
+    console.log('going to call civil service');
     await this.client.post(CIVIL_SERVICE_CASES_URL,{ match_all: {} }, config)
       .then(response => {
         claims = response.data.cases.map((claim: CivilClaimResponse) => Object.assign(new Claim(), claim.case_data));
       }).catch(error => {
+        console.log('I am here');
```
Comment (TatsyMaslova) [inline src/main/app/client/civilServiceClient.ts:36]: Not sure whether we need this here

Comment (sampankumar) [reply src/main/app/client/civilServiceClient.ts:36]: Just adding some logs to see if the code is reaching here or not

Comment (TatsyMaslova) [reply src/main/app/client/civilServiceClient.ts:36]: Ah ok. perhaps though this console log message should not be in the final version that goes to master? we could log the error instead

Comment (sampankumar) [reply src/main/app/client/civilServiceClient.ts:36]: Will remove these later.

Comment (TatsyMaslova) [reply src/main/app/client/civilServiceClient.ts:36]: okie

**Review by TatsyMaslova — APPROVED** [review]

**Comment by mounikahmcts** [comment]
Merging this with console logs for now. The code should be refactored with logger later.

**Pull request merged**